core/templates/pages/learner-dashboard-page/learner-dashboard-page.component.ts

```diff
@@ -380,6 +382,14 @@ angular.module('oppia').component('learnerDashboardPage', {
         return DateTimeFormatService.getLocaleAbbreviatedDatetimeString(
           millisSinceEpoch);
       };
+
+      ctrl.addFocusWithoutScroll = function(label) {
+        FocusManagerService.setFocus(label);
+        setTimeout(function() {
+          window.scrollTo(0, 0);
+        }, 5);
```

Comment:
ditto: auto scrolling

Reply:
if we autoscroll on small screen it will be something like this when the page loads:-
![Screenshot from 2021-03-17 13-16-32](https://user-images.githubusercontent.com/68547101/111432679-56839080-8723-11eb-846a-7befd62648f9.png)
By preventing to autoscroll:
![Screenshot from 2021-03-17 13-19-19](https://user-images.githubusercontent.com/68547101/111432735-68653380-8723-11eb-926a-711e9d9ec4cc.png)
Button is still in focus and visible.